addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -387,7 +387,7 @@ patch(Chatter.prototype, {
         this.props.record?.load();
     },
 
-    onFollowerChanged(thread) {
+    onFollowerChanged() {
         document.body.click(); // hack to close dropdown
         this.reloadParentView();
```

Comment:
Technically the param was still useful but the code was lacking a use for it.

In particular, if the followers changed from a previous `thread` (keeping in mind all of this code is async), there should be no need to reload the newest thread if it is different.

Reply:
I don't understand what you are trying to explain.
This param is not used in the function nor inside any override. Does the fact that thread is a param trigger any side effect? Or we are missing the code of this effect?

Reply:
We are missing a condition based on thread to avoid unnecessary reload if the thread has changed since the trigger.

click follow -> start slow rpc -> change thread (pager) -> return of rpc -> unnecessary reload on new thread